<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AngelWindow</class>
 <widget class="QWidget" name="AngelWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>0</y>
     <width>410</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Welcome to find angel calculator!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Let's select a type of figures.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choose_figure1">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>100</y>
     <width>410</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>vector</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>plate</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="choose_figure2">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>135</y>
     <width>410</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>vector</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>plate</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="input_figure2" native="true">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>500</y>
     <width>650</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="form_figure1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>by point and vector</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by classic form</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by canonical form</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by parametric form</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="form_figure2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>by three points</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by line and point</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by classic form</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="input_figure1" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>260</y>
     <width>650</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="answer">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>650</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="form_subfigure2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>455</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>by point and vector</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by classic form</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by canonical form</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by parametric form</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="output">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>680</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="form_subfigure1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>215</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>by point and vector</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by classic form</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by canonical form</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>by parametric form</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>415</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;second figure:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>175</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;first figure:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>choose_figure1</zorder>
  <zorder>choose_figure2</zorder>
  <zorder>form_figure1</zorder>
  <zorder>input_figure1</zorder>
  <zorder>form_figure2</zorder>
  <zorder>input_figure2</zorder>
  <zorder>answer</zorder>
  <zorder>form_subfigure2</zorder>
  <zorder>output</zorder>
  <zorder>form_subfigure1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
